_posts/2017-4-12-swarm-service-discovery.markdown

```diff
@@ -96,7 +94,7 @@ Let's list the tasks of the wordpressdb service.
 docker service ps wordpressdb
 ```
 
-You should get an output like the following one where the 2 tasks (replicas) of the service are listed.
+You should get an output like the following one where the 1 task  of the service are listed.
```

Comment:
This sentence needs correction:

`You should get an output like the following one where 1 task of the service is listed`